### PR TITLE
upgrade kafka-node

### LINKED ISSUE #24: Upgrade kafka-node
This transitively pulls in vulnerable minimatch (https://nodesecurity.io/advisories/118). Any chance you could upgrade?

https://github.com/openzipkin/zipkin-js/tree/master/packages/zipkin-transport-kafka

Doesn't seem like there's any breaking changes, should be straight forward: https://github.com/SOHU-Co/kafka-node/blob/master/CHANGELOG.md

### PR DESCRIPTION
**Pull request by sveisvei**
closes #24.

There should be only fixes here, so a minor jump is fine.
https://github.com/SOHU-Co/kafka-node/blob/master/CHANGELOG.md

### CODE REVIEW

File: packages/zipkin-transport-kafka/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "dependencies": {
-    "kafka-node": "^0.3.2"
+    "kafka-node": "^0.5.9"
```
Comment (SimenB) [inline packages/zipkin-transport-kafka/package.json]: `"^0.5.0"`? Internally we depend on `0.5.3` locked (although that might be ok to bump)

**Comment by adriancole** [comment]
on the note of releases, I noticed we've been bumping patch versions for things that seem like they should probably be minor versions. I know pre 1.0 versions don't say much, but should we start bumping minors for stuff like this? feels fair.

File: packages/zipkin-transport-kafka/package.json (hunk 1 above)
Comment (SimenB) [inline packages/zipkin-transport-kafka/package.json]: `"^0.5.0"`? Internally we depend on `0.5.7` locked (although that might be ok to bump)

**Comment by sveisvei** [comment]
I feel this is a minor indeed. 👍

**Pull request merged**